src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -828,14 +828,22 @@ private async Task<string> CreatePullRequestAsync(List<UpdateAssetsParameters> u
             }
         }
 
-        private void UpdatePRDescriptionDueConfigFiles(List<GitFile> commitedFiles, StringBuilder globalJsonSection)
+        public static void UpdatePRDescriptionDueConfigFiles(List<GitFile> committedFiles, StringBuilder globalJsonSection)
```

Comment:
Wondering about the visibility change?

Reply:
Haha, oops. I did that when I was testing something. I'll revert.

Reply:
Yep, basically what I figured haha